Add Windows service `trigger_start` filter to config (#15975)
DataDog/integrations-core: windows_service/README.md

- `automatic`
- `automatic_delayed_start`

Beginning with Agent version 7.50, the check can select which services to monitor based on whether they have a [Service Trigger assigned][17].
Below are some examples showing possible configurations.
```yaml

> # Matches all services that does not have a trigger

Reviewer: ```suggestion
# Matches all services that do not have a trigger
```

services:
  - trigger_start: false

> # Matches all services with an automatic startup type and has a trigger

Reviewer 2: ```suggestion
# Matches all services with an automatic startup type and excludes services with triggers
```
